avm/res/api-management/service/tests/e2e/max/dependencies.bicep

```
@@ -4,13 +4,219 @@ param managedIdentityName string
 @description('Optional. The location to deploy resources to.')
 param location string = resourceGroup().location
 
+@description('Required. The name of the Public IP to create.')
+param publicIPName string
+
+@description('VNet name')
+param vnetName string = 'VNet'
+
+@description('Address prefix')
+param vnetAddressPrefix string = '10.0.0.0/16'
+
+@description('Subnet Prefix')
+param subnetPrefix string = '10.0.0.0/24'
+
+@description('Subnet Name')
+param subnetName string = 'Subnet'
```

This and the next is just a recommendation to align with other test cases. Feel free to reject if you prefer the original implementation 😉 

```suggestion
@description('Required. The name of the Virtual Network to create.')
param virtualNetworkName string

var addressPrefix = '10.0.0.0/16'
```